src/zenml/artifacts/utils.py

```diff
@@ -272,29 +245,113 @@ def _create_version() -> Optional[ArtifactVersionResponse]:
         )
 
     if manual_save:
+        _link_artifact_version_to_model_version_in_context(
+            name=name,
+            response=response,
+            is_model_artifact=is_model_artifact,
+            is_deployment_artifact=is_deployment_artifact,
+        )
+
+    return response
+
+
+def link_existing_data_as_artifact(
```

Comment:
Maybe we could shorten this to something like `register_artifact(...)`? Or do you think that would not be clear enough in what it does?

Reply:
Doch, `register_artifact(...)` reflects it well, I will switch

Reply:
Renamed